modules/lang-painless/src/main/java/org/elasticsearch/painless/antlr/Walker.java

```diff
@@ -627,12 +629,25 @@ public ANode visitBool(BoolContext ctx) {
         } else if (ctx.BOOLOR() != null) {
             operation = Operation.OR;
         } else {
-            throw location(ctx).createError(new IllegalStateException("Illegal tree structure."));
+            throw location(ctx).createError(new IllegalStateException("illegal tree structure"));
         }
 
         return new EBool(location(ctx), operation, left, right);
     }
 
+    @Override
+    public ANode visitElvis(ElvisContext ctx) {
```

Comment:
Why was this moved?

Reply:
Rules got moved around here to stay in line with the ordering in the grammar.